src/lib/parts.ts

```diff
@@ -128,6 +132,14 @@ export class AttributePart implements Part {
   }
 }
 
+/**
+ * A Part that controls a location within a Node tree. Like a Range, NodePart
+ * has start and end locations and can set and update the Nodes between those
+ * locations.
+ *
+ * NodeParts support several value types: primitives, Nodes, TemplateResults,
```

Comment:
```suggestion
 * NodeParts support several value types: primitives, Nodes, TemplateResults, and other NodeParts,
```

Reply:
this isn't quite correct, see other comment